src/handlers/swap_handlers.ts

```diff
@@ -142,34 +190,6 @@ export class SwapHandlers {
             throw new InternalServerError(e.message);
         }
     }
-    // tslint:disable-next-line:prefer-function-over-method
-    public async getSwapTokensAsync(_req: express.Request, res: express.Response): Promise<void> {
```

Comment:
just to confirm:`getSwapTokensAsync` and `getTokenPricesAsync` just got moved, but didn't change. is that correct?

Reply:
Yes, that is correct.  I tried to explain that in the commit message for https://github.com/0xProject/0x-api/pull/172/commits/66c9d9dd3aa71b96b2a3b85000f9c0a44bc0bffd, but it was a complicated write-up...